.github/workflows/ci-sharktank.yml

```diff
@@ -36,15 +36,14 @@ jobs:
     env:
       PIP_CACHE_DIR: "${{ github.workspace }}/.pip-cache"
     steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
```

Comment:
Same here, there is another job in this workflow that needs to be updated as well.